Python/Python.sublime-syntax

```diff
@@ -342,34 +519,27 @@ contexts:
             - match: (?=\))
               pop: true
             - include: keyword_arguments
+            - match: ','
+              scope: punctuation.separator.parameters.python
+            - include: inline_for
             - include: expressions
 
   lambda:
-    - match: \b(lambda)(?=\s+)
+    - match: \b(lambda)
       scope: storage.type.function.inline.python
       push:
         - meta_scope: meta.function.inline.python
+        - meta_content_scope: meta.function.inline.parameters.python
+        - include: line_continuation_or_pop
         - match: '\:'
           scope: punctuation.section.function.begin.python
           pop: true
-        - match: \s+
-          push:
-            - meta_content_scope: meta.function.inline.parameters.python
-            - match: (?=\:)
-              pop: true
-            - include: keyword_arguments
-            - match: '({{identifier}})\s*(?:(,)|(?=[\n\)\:]))'
-              captures:
-                1: variable.parameter.python
-                2: punctuation.separator.parameters.python
-
-  late-keywords:
-    - match: \b(def|lambda)\b
-      scope: storage.type.function.python
-    - match: \b(class)\b
-      scope: storage.type.class.python
-    - match: \b(async)\b
```

Comment:
Generally it is nice to not have colors jumping around as the user is typing, so there are situations where we highlight things that aren't valid constructs. Trying to have the syntax provide linter-like feedback usually just leads to a less than desirable editing experience.